docs/src/main/tut/faq.md

```diff
@@ -8,7 +8,7 @@ position: 4
 # Frequently Asked Questions
 
 ## Questions
-
+ * [What is the difference between cats and scalaz](#diff-scalaz) 
```

Comment:
Maybe this shouldn't be the first item in the list. I think that a lot of people come to Cats without familiarity with Scalaz, so something like "What imports do I need" would probably be a more straightforward first item for them than introducing another FP library.